scene/gui/color_picker.cpp

```diff
@@ -507,8 +508,9 @@ void ColorPicker::_focus_exit() {
 }
 
 void ColorPicker::_html_focus_exit() {
-	_html_entered(c_text->get_text());
-	_focus_exit();
+	//Commented out as a temporary fix for issue #21860. So that raw mode can work properly until raw mode settings are saved/loaded from the scene.
+	//_html_entered(c_text->get_text());
+	//_focus_exit();
```

Comment:
A proper fix is better than a temporary work-around. Perhaps `if (!raw_mode_enabled) {` should be used here too?

Reply:
It can't be at the moment as the raw mode is not saved permanently so as soon as you switch nodes it doesn't work. I'd add the ability to permanently save raw mode status but that is beyond my abilities. As no one else who could fix it better has shown interest nor has their been a reply from the person who caused the issue to began with I figured I'd at lest fix it for the moment as it's quite an annoying issue.